scanner/scanner.go

```diff
@@ -0,0 +1,185 @@
+package scanner
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"math"
+	"time"
+
+	"github.com/SkynetLabs/malware-scanner/clamav"
+	"github.com/SkynetLabs/malware-scanner/database"
+	"github.com/SkynetLabs/skynet-accounts/build"
+	"github.com/sirupsen/logrus"
+	"gitlab.com/NebulousLabs/errors"
+)
+
+var (
+	// sleepBetweenScans defines how long the scanner should sleep after
+	// scanning the DB and not finding any skylinks to scan.
+	sleepBetweenScans = build.Select(
+		build.Var{
+			Dev:      30 * time.Second,
+			Testing:  100 * time.Millisecond,
+			Standard: 10 * time.Second,
```

Comment:
I would swap Dev and Standard. Dev is usually between testing and standard.

Reply:
I have chosen that value because it was a convenient value to use while developing. I would run the service and watch the scans go by. I want that to happen slower that it would do in prod.

I stand by the value. It's what made sense to me while using it.